src/pages/Hero.tsx

```diff
@@ -27,10 +25,6 @@ export const Hero = () => {
     const { hideHero, setHideHero, t, denomination, separator } =
         useGlobalContext();
 
-    const openNodeInfo = async () => {
-        window.open(`${config.lightningExplorerUrl}/${nodePubkey()}`, "_blank");
```

Comment:
This was the only place we used `lightningExplorerUrl`. Also remove it from `config.ts`

Reply:
Done